client/my-sites/stats/stats-list/stats-list-actions.jsx

```diff
@@ -96,15 +79,14 @@ function useActionItems( { data, moduleName } ) {
  * Render a list of `action` icons based on action array from a list item.
  * Possible types: External Link redirect, Specific page statistics redirect, Spam, Promote, Follow.
  */
-const StatsListActions = ( { data, moduleName, children } ) => {
+const StatsListActions = ( { data, moduleName, children, isMobileMenuVisible, onClick } ) => {
```

Comment:
Would it be worth considering a more intuitive name for `onClick`?

Reply:
Sure! Would `onMobileMenuClick` be a better name?

Reply:
Sounds good!

Reply:
Done!